Step2/Install_Win_Features.ps1

```diff
@@ -0,0 +1,86 @@
+# Import the configuration
+Import-Module -Name .\Config.ps1
+
+# Function to handle logging and console output
+function Log-Message {
+    param (
+        [string]$Message
+    )
+    
+    $Timestamp = (Get-Date).ToString("yyyy-MM-dd HH:mm:ss")
+    "$Timestamp - $Message" | Out-File -FilePath $Global:LogFilePath -Append
+    Write-Host $Message
+}
+
+function Install_Win_Features {
+    param (
+        [string[]]$Features

```

Comment:
Here you parameter is set to type StringList and not Array

Reply:
It defines an array of strings. 
This is the correct type to use for an array of strings, and it is equivalent to specifying an array explicitly.